packages/features/settings/OutOfOfficeEntriesListForTeam.tsx

```diff
@@ -0,0 +1,280 @@
+import { keepPreviousData } from "@tanstack/react-query";
+import type { ColumnDef } from "@tanstack/react-table";
+import { Trans } from "next-i18next";
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
+import { useFormState } from "react-hook-form";
+
+import dayjs from "@calcom/dayjs";
+import { getUserAvatarUrl } from "@calcom/lib/getAvatarUrl";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Avatar, Button, DataTable, EmptyScreen, Icon, showToast, SkeletonText, Tooltip } from "@calcom/ui";
+import type { BookingRedirectForm } from "@calcom/web/pages/settings/my-account/out-of-office";
+
+interface OutOfOfficeEntry {
+  id: number;
+  uuid: string;
+  start: Date;
+  end: Date;
+  toUserId: number | null;
+  toUser: {
+    username: string;
+  } | null;
+  reason: {
+    id: number;
+    emoji: string;
+    reason: string;
+    userId: number;
+  } | null;
+  notes: string | null;
+  user: { id: number; avatarUrl: string; username: string; email: string } | null;
+}
```

Comment:
This is used twice, can you pls reuse it another component?

Reply:
yeah, it can be reused, will update